google/oauth2/credentials.py

```diff
@@ -90,6 +91,17 @@ def client_secret(self):
         return self._client_secret
 
     @property
+    def refresh_grant_response(self):
```

Comment:
Yep, I actually found a use case where this is necessary. When you refresh a token, you actually get both an access token and an id token back in the response. The other option is to add an `id_token` field to `oauth2.Credentials` and only populate that, but it'd have the same caveats as this property. I'm cool with either, and the `id_token` approach has some benefits.